docs/elements.md

```diff
@@ -998,8 +998,13 @@ def generate(data):
 | `ignore-case`             | boolean             | false    | Whether or not to enforce case sensitivity (e.g. "hello" != "HELLO").                                                                                                     |
 | `normalize-to-ascii`      | boolean             | false    | Whether non-English characters (accents, non-latin alphabets, fancy quotes) should be normalized to equivalent English characters before submitting the file for grading. |
 | `placeholder`             | text                | None     | Hint displayed inside the input box describing the expected type of input.                                                                                                |
-| `size`                    | integer             | 35       | Size of the input box.                                                                                                                                                    |
+| `size`                    | integer             | 35       | Width of the input box.                                                                                                                                                   |
 | `show-help-text`          | boolean             | true     | Show the question mark at the end of the input displaying required input parameters.                                                                                      |
+| `multiline`               | boolean             | false    | Whether or not to to allow the input to include line breaks.                                                                                                              |
+
+#### Details
+
+The student answer will use a single `\n` as a newline character, even if `multiline` is not enabled. To avoid issues with newlines, they can be ignored by enabling `remove-spaces`. Keep in mind that longer strings may not display cleanly on the submitted or correct answer panels, so it may be best to hide these panels if students are submitting longer strings.
```

Comment:
This is more specific to technical implementation than useful information for instructors. I don't think mentioning the `\n` here is necessary, what is more important is to ensure instructors are aware of issues with terminating line breaks.

About hiding submitted/correct answers, I don't think that's the solution. I'll discuss that in a separate comment.

Here is a suggestion.

```suggestion
#### Using multiline inputs

Note that, in multiline inputs, it can be hard to distinguish between inputs with or without a terminating line break (i.e., an additional "Enter" at the end of the input). Because of that, you are strongly encouraged to use `remove-leading-trailing="true"` when using multiline inputs.
```

About `remove-spaces`: this might require a bit of thought. Should line breaks be counted as spaces? This may depend on context, but I would assume that the default should be no. In that case, should the leading/trailing handling be different in this consideration?

Reply:
Line breaks are already being counted as spaces (the `remove-spaces` attribute gets rid of all whitespace).